internal/reliabletransport/receiver.go

```diff
@@ -3,7 +3,6 @@ package reliabletransport
 import (
 	"bytes"
 	"fmt"
-	"log"
```

Comment:
Were were we previously using "log" in this file? Maybe we need to delete more lines?

Reply:
it seems it was only a bogus import left